src/.eslintrc.json

```diff
@@ -1,4 +1,8 @@
 {
+	"env": {
+		"browser": false,
```

Comment:
What do you mean by "these files only"?

The configs cascade down in ESLint. If you don't specify `browser: false`, you depend on the parent config to not specify `browser: true`. Since we have to reset the `node` env, I wanted to specify the browser one explicitly, it seems less fragile.